### DISCUSSION TITLE
Click on element at specific coordinates

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [go-rod/rod#869] [General]
I was wondering if there was a method to click on an html element at specific coordiantes. 

The element that I want to click on is a canvas with many different areas and I calculate the location that should be clicked beforehand.

Forgive me if I'm wrong but I don't think I've seen any documentation on that

**Comment by ysmood** [comment]
https://github.com/go-rod/rod/issues/666

**Reply by FrenchGithubUser** [reply]
That's it indeed, I couldn't find this issue. Thanks.
